editpost.php

```diff
@@ -938,6 +943,17 @@
 		}
 	}
 
+	$php_max_upload_filesize = return_bytes(ini_get('max_upload_filesize'));
+	$php_post_max_size = return_bytes(ini_get('post_max_size'));
+
+	if ($php_max_upload_filesize != 0 && $php_post_max_size != 0)
```

Comment:
Our coding style enforces braces (`{`, `}`) for all statements, would be great if you could add them here 😄

Reply:
Sorry for all the lacking braces, I am used to other coding style :P